lib/main.dart

```diff
@@ -128,13 +140,10 @@ class _MyHomePageState extends State<MyHomePage> {
                     emailField,
                     SizedBox(height: 25.0),
                     passwordField,
-                    SizedBox(
-                      height: 35.0,
-                    ),
+                    SizedBox(height: 25.0,),
                     loginButton,
-                    SizedBox(
-                      height: 15.0,
-                    ),
+                    SizedBox(height: 65.0,),
+                    registerButton,
```

Comment:
For demo and debugging, adds two buttons to bypass the login screen and its call to the database. 

```suggestion
                    registerButton,
                   
                   /*For demo purposes, these bypass the loginButton*/
                  SizedBox(height: 30,),
                  Row(
                    mainAxisAlignment: MainAxisAlignment.end,
                    children: <Widget>[
                      RaisedButton(
                        padding: EdgeInsets.fromLTRB(0, 0, 0, 0),
                        child: Text('demo: S'),
                        onPressed: (){
                          Navigator.push(
                            context,
                            MaterialPageRoute(builder: (context) => StudentScreen()),
                          );
                        },
                      ),
                      RaisedButton(
                        padding: EdgeInsets.fromLTRB(0, 0, 0, 0),
                        child: Text('demo: M'),
                        onPressed: (){
                          Navigator.push(
                          context,
                          MaterialPageRoute(builder: (context) => MentorScreen()),
                          );
                        },
                      ),
                   ],
                  ),
```